llvm/test/CodeGen/AArch64/sve-intrinsics-perm-select.ll

```diff
@@ -1,6 +1,7 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
 ; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sve < %s | FileCheck %s --check-prefixes=CHECK,SVE
 ; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sve2 < %s | FileCheck %s --check-prefixes=CHECK,SVE2
+; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sme2p2,+sme -force-streaming < %s | FileCheck %s --check-prefixes=CHECK,SSVE
```

Comment:
I don't think you need this change. `llvm/test/CodeGen/AArch64/sve2p2-intrinsics.ll` contains the same intrinsic tests for `llvm.aarch64.sve.compact` and already includes a RUN line  for +sme2p2.